ubuntu/16.04/usr/local/share/container/baseimage-10.sh

```diff
@@ -23,7 +23,7 @@ do_build() {
 }
 
 check_development_start() {
-  if [ "$DEVELOPMENT_MODE" -eq 0 ]; then
+  if [ "$DEVELOPMENT_MODE" == 'true' ]; then
```

Comment:
YAML have booleans. But I think that it's converted as `0` & `1` when injected through environment variables by k8s.